trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    return @nameable.correct_name.strip if @nameable.correct_name.length > 10
+
+    @nameable.correct_name
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=base%20Decorator%20class.-,Implement%20a%20method%20correct_name%20that%20makes%20sure%20that%20the%20output%20of%20%40nameable.correct_name%20has%20a%20maximum%20of%2010%20characters.%20If%20it%27s%20longer%20it%20should%20trim%20the%20word.,-See%20your%20decorators), Please makes sure that the output of `@nameable.correct_name` has a maximum of 10 characters because now it returns the whole name without change.

![image](https://github.com/danielsafari143/school/assets/68796232/4b2acd5d-52d6-4801-872f-d30b6e8074bc)

[Hint] you can make it like that:
```ruby
 def correct_name
    @nameable.correct_name[0..9]
 end
```